camelot/admin/action/list_filter.py

```diff
@@ -935,3 +767,182 @@ def model_run(self, model_context, mode):
         yield action_steps.SetFilter(self, value)
 
 search_filter = SearchFilter()
+
+
+@dataclass
+class FilterMode(Mode):
+
+    checked: bool = False
+
+    def __init__(self, value, verbose_name, checked=False):
+        super(FilterMode, self).__init__(name=value, verbose_name=verbose_name)
+        self.checked = checked
+
+    def decorate_query(self, query, value):
+        return self.decorator(query, value)
+
+# This used to be:
+#
+#     class All(object):
+#         pass
+#
+# It has been replaced by All = '__all' to allow serialization
+#
+All = '__all'
+
+class Filter(Action):

```

Comment:
heeft deze geen render hint nodig ?  of w die nooit as such gebruikt ?

Reply:
Nee idd, op dit moment niet voor zover ik weet.